var/spack/repos/builtin/packages/mpc/package.py

```diff
@@ -25,12 +25,12 @@
 from spack import *
 
 
-class Mpc(AutotoolsPackage):
+class Mpc(Package):
```

Comment:
your PR is certainly not based off the current `develop`. This should stay as `AutotoolsPackage` without extra `install` method.